### DISCUSSION TITLE
Ice masks in models

### DISCUSSION THREAD

**Discussion by NicolajHansen90** [IACS-cryo/Delineation-WG#31] [Modeling]
Hi fellow modellers 

I have been thinking about how best to implement a new ice mask into models and make sure that we all use the same ice mask. I have some questions that I would like to discuss/ get your opinions on:
1: How do we deal we different model resolutions? Should we make different ice masks in the most common resolutions for both ice sheets, that people can use? Or should we make high-resolution shapefile and provide a script, so each modelling group can make their own but still regridded using the same method?
2: How do we convince people to use the new ice mask? my personal fear is that people will just keep doing what they always have done due to convenience. 
3: Should we just use the new mask from now on, or do we want to rerun old simulations?
4: While we create this new ice mask, should we then also select a common topography?

There are probably other important questions, you are welcome to add more questions.

**Comment by hgoelzer** [comment]
Hi, could we distinguish in this discussion between different modelling efforts? SMB, ice sheets, glaciers, ... 
I guess you are looking at this from the large-scale SMB/RCM perspective. For ice sheet modellers the masking problem is also important, but has other issues. As boundary condition, ISMs may require SMB to be defined outside of the observed mask. So I would hope SMB modellers do not cut off their prognostic products too tightly. Diagnostically is another question, of course.

**Reply by NicolajHansen90** [reply]
Good idea! We should take all modelling types into account. Yes, I am working with SMB/RCM, most of us (I think) in the SMB/RCM world use ice masks that are non-evolving/constant over time. I guess that is different for ISM. Furthermore, most SMB models, to my knowledge, only compute the SMB over the ice mask, so if ISM needs the SMB outside the ice mask, we probably need to discuss a procedure for that?
